kubernetes/loculus/values.yaml

```diff
@@ -1850,6 +1850,11 @@ defaultOrganisms:
           <<: *preprocessingConfigFile
           log_level: INFO
           nextclade_dataset_name: community/hodcroftlab/enterovirus
+          nextclade_dataset_name_map:
+            CV-A16: "community/hodcroftlab/enterovirus/cva16"
+            CV-A10: "community/hodcroftlab/enterovirus/cva10"
+            EV-A71: "community/hodcroftlab/enterovirus/eva71"
+            EV-D68: "community/hodcroftlab/enterovirus/evd68"
```

Comment:
@anna-parker I still have this PR open. Do you know whether this is still relevant? Or did you do those changes in the meantime already?